### DISCUSSION TITLE
[460790] 猎天使魔女 Bayonetta [本体]

### DISCUSSION THREAD

**Discussion by yige666s** [pjy612/SteamManifestCache#408] [Share]
`appid:460790`

keys:

```
460792;e80066b74e53039c621ea6adf902751eb8f1dac4ca0d402199191086cf6302b7
```

code: BBDB9D28EC146217
email : [yige29805@gmail.com](mailto:yige29805@gmail.com)
[460792_3254100067932476528.zip](https://github.com/pjy612/SteamManifestCache/files/13998280/460792_3254100067932476528.zip)

**Comment by ZEERDEER** [comment]
* 标题格式 #26

**Reply by yige666s** [reply]
已修改
